src/Uno.Extensions.Reactive/Utils/Debugging/DebugConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using Microsoft.Extensions.Logging;
+
+namespace Uno.Extensions.Reactive.Utils.Debugging;
+
+internal class DebugConfiguration
+{
+	private static bool? _isDebugging;
+
+	public static bool IsDebugging => _isDebugging ??= Debugger.IsAttached | Logging.LogExtensions.Log<ISignal>().IsEnabled(LogLevel.Debug);
```

Comment:
Not a large difference, but is the single arithmetic `|` what was intended?

Reply:
Indeed not intended nor desirable ... let fix it.
```suggestion
	public static bool IsDebugging => _isDebugging ??= Debugger.IsAttached || Logging.LogExtensions.Log<ISignal>().IsEnabled(LogLevel.Debug);
```